.air.toml

```diff
@@ -0,0 +1,50 @@
+root = "."
+tmp_dir = "tmp"
+
+[build]
+  args_bin = ["api", "consumer", "instrumentation", "mock_rbac"]
+  bin = "./release/content-sources"
+  cmd = "make build"
```

Comment:
You might change this to 
```go build -o ./release/content-sources cmd/content-sources/main.go```

so then its only rebuilding the binary used by the server.  That will speed it up a lot!

Reply:
changed! thanks for the tip 👌🏼